legal-api/src/legal_api/services/permissions.py

```diff
@@ -111,6 +133,8 @@ def get_authorized_user_role(token_info: dict = None) -> str:
         roles_in_token = token_info.get('realm_access', {}).get('roles', [])
         for role in role_priority:
             if role in roles_in_token:
+                if role == authz.SYSTEM_ROLE:
+                    role = authz.STAFF_ROLE
```

Comment:
STAFF does not have all permissions. Should SYSTEM have all possible permissions? If yes then this needs to be changed.